.github/CODEOWNERS

```diff
@@ -16,8 +17,81 @@
+
+/xml/System.Buffers**/** @layomia @JeremyKuhne
+/xml/System.CodeDom**/** @buyaa-n @krwq
+/xml/System.Collections**/** @eiriktsarpalis @layomia
+/xml/System.ComponentModel**/** @JeremyKuhne @safern
+/xml/System.ComponentModel.Composition**/** @maryamariyan @ViktorHofer
+/xml/System.ComponentModel.DataAnnotations**/** @lajones @ajcvickers
+/xml/System.Composition**/** @maryamariyan @ViktorHofer
+/xml/System.Configuration**/** @maryamariyan @safern
+/xml/System.Data**/** @ajcvickers @cheenamalhotra @david-engel
+/xml/System.Data.SqlClient**/** @cheenamalhotra @david-engel @karinazhou @JRahnama
+/xml/System.Diagnostics**/** @tommcdon @krwq
+/xml/System.Diagnostics/DiagnosticSource.xml @Anipik
+/xml/System.Diagnostics/EventLog*.xml @Anipik
+/xml/System.Diagnostics/PerformanceCounter*.xml @Anipik
+/xml/System.Diagnostics/Process*.xml @adsitnik @eiriktsarpalis
+/xml/System.Diagnostics/TraceSource.xml @Anipik
+/xml/System.Diagnostics.Tracing**/** @Anipik
+/xml/System.DirectoryServices**/** @tquerec @josephisenhour
+/xml/System.Drawing**/** @JeremyKuhne @maryamariyan
+/xml/System.Dynamic**/** @cston @333fred
+/xml/System.Globalization**/** @safern @krwq @tarekgh
+/xml/System.IO.Compression**/** @carlossanlop @ericstj
+/xml/System.IO.Pipelines**/** @davidfowl @halter73 @jkotalik @anurse
+/xml/System.IO**/** @JeremyKuhne @carlossanlop @jozkee
+/xml/System.Linq**/** @eiriktsarpalis
+/xml/System.Linq/Parallel*.xml @tarekgh @kouvel
+/xml/System.Linq.Expressions**/** @cston @333fred
+/xml/System.Management**/** @Anipik
+/xml/System.Net**/** @dotnet/ncl
+/xml/System.Numerics**/** @tannergooding @prgovind
+/xml/System.Reflection**/** @steveharter @GrabYourPitchforks
+/xml/System.Reflection.Emit**/** @steveharter @GrabYourPitchforks
+/xml/System.Reflection.Metadata**/** @tmat @nguerrera
+/xml/System.Resources**/** 	@buyaa-n @tarekgh @krwq
+/xml/System.Runtime**/** @bartonjs @joperezr
+/xml/System.Runtime.Caching**/** @StephenMolloy @HongGit
+/xml/System.Runtime.CompilerServices**/** @Anipik @steveharter
+/xml/System.Runtime.InteropServices**/** @AaronRobinsonMSFT @jkoritzinsky
+/xml/System.Runtime.Intrinsics**/** @tannergooding @CarolEidt @RussKeldorph
+/xml/System.Security**/** @bartonjs @GrabYourPitchforks @krwq
+/xml/System.ServiceModel.Syndication**/** @StephenMolloy @HongGit @mconnew
+/xml/System.ServiceProcess**/** @Anipik
+/xml/System.Text**/** @tarekgh @GrabYourPitchforks
+/xml/System.Text/Encoding*.xml @layomia @krwq @tarekgh
+/xml/System.Text.Encodings.Web**/** @GrabYourPitchforks @layomia @tarekgh
+/xml/System.Text.Json**/** @layomia @steveharter @jozkee
+/xml/System.Text.RegularExpressions**/** @prgovi @eerhardt
+/xml/System.Threading**/** @kouvel
+/xml/System.Threading.Channels**/** @tarekgh @stephentoub
+/xml/System.Threading.Tasks**/** @tarekgh @stephentoub
+/xml/System.Transactions**/** @dasetser @HongGit
+/xml/System.Web**/** @scottaddie
```

Comment:
The owner of System.Web is not mentioned in the area owners document, but I'll keep it here unless there's an objection.

Reply:
@scottaddie are you still the right person?

Reply:
@danmosemsft You can keep me there for System.Web.